### DISCUSSION TITLE
move images by dimension in a bat file

### DISCUSSION THREAD

**Discussion by itssofix** [ImageMagick/ImageMagick#4357] [Digital Image Processing] [answered]
Hello guys,

I've been trying to move photos with imagick for a while now. the idea behind it is to separate low res (below 500X500) and high res photos. I can currently get hold of the dimensions per photo, but after that it keeps getting stuck.

this is the code of the width as an example, but i am not getting it in a string or variable. Can someone help me to find a solution? I've tried many times but it always fails

magick identify -format "%w" image.jpg

**Comment by fmw42** [comment]
Check out PHP getImagesize() at https://www.php.net/manual/en/function.getimagesize.php. It should do what you want to get dimensions.

**Reply by itssofix** [reply]
thanks for the link, but it was specifically meant for the cmd windows environment

**Comment by snibgo** [comment, marked as answer]
> I've been trying to move photos with imagick ...

Do you mean IMagick, which is used in PHP as an interface to ImageMagick? If so, see IMagick documentation.

But your question seems to be about how to get an image width into a Windows CMD shell envivironment variable.
```
for /F "usebackq" %W in (`magick identify -format %w toes.png`) do set mywidth=%W
```

**Reply by itssofix** [reply]
It is indeed with windows cmd. The program is fully finished and working, thanks to your code :)
